src/js/base/api.js

```diff
@@ -250,20 +250,23 @@ ripe.Ripe.prototype._getQuery = function(options) {
     return this._buildQuery(options.params);
 };
 
-ripe.Ripe.prototype._getConfigOptions = function(options, brand, model, variant) {
+ripe.Ripe.prototype._getConfigOptions = function(options) {
     options = options || {};
-    brand = brand || this.brand;
-    model = model || this.model;
-    variant = variant || this.variant;
-    var params = {};
+    var brand = options.brand === undefined ? this.brand : options.brand;
+    var model = options.model === undefined ? this.model : options.model;
     var url = this.url + "brands/" + brand + "/models/" + model + "/config";
-    if (variant) {

```

Comment:
no variant ?

Reply:
no `variant` involved on this API, https://docs.platforme.com/#config